docs/src/main/sphinx/release/release-355.md

```diff
@@ -0,0 +1,52 @@
+# Release 355 (XX Apr 2021)
+
+## General
+
+* Report tables that are directly referenced by query in `QueryCompletionEvent`. ({issue}`7330`)
+* Report columns that are the target of `INSERT` or `UPDATE` queries in `QueryCompletionEvent`. This includes
+  information about which input columns they are derived from. ({issue}`7425`, {issue}`7465`)  
+* Rename `optimizer.plan-with-table-node-partitioning` config property to `optimizer.use-table-scan-node-partitioning`. ({issue}`7257`)
+* Improve query parallelism when table bucket count is small compared to number of nodes. 
+  This optimization is now triggered automatically when the ratio between table buckets and 
+  possible table scan tasks exceeds or is equal to `optimizer.table-scan-node-partitioning-min-bucket-to-task-ratio`. ({issue}`7257`)
+* Include information about {doc}`/admin/spill` in {doc}`/sql/explain-analyze`. ({issue}`7427`)
+* Disallow inserting data into tables that have row filters. ({issue}`7346`)
```

Comment:
```suggestion
* Disallow inserting data into a table that has a row filter. ({issue}`7346`)
```

?

Reply:
The plural form is fine. It's a reference to tables and row filters in the general sense.

Reply:
Yes, this reads OK. It would be great to have a guideline whether we prefer plural (general sense) or singular forms.